src/Propel/Runtime/ActiveQuery/BaseModelCriteria.php

```diff
@@ -261,6 +266,22 @@ public function getTableMap()
         return $this->tableMap;
     }
 
+    /**
+     * Returns the name of the table as used in the query.
+     *
+     * Either the SQL name or an alias.
+     *
+     * @return string
+     */
+    public function getTableNameInQuery()
+    {
+        if ($this->useAliasInSQL && !empty($this->modelAlias)) {
```

Comment:
Can we use non cloaking

    && $this->modelAlias

instead?

Reply:
Yes, good point. Changed it